packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -266,6 +269,13 @@ export class PKPBase<T = PKPBaseDefaultParams> {
    * @throws {Error} - Throws an error if `pkpPubKey` is not provided, if `controllerAuthSig` or `controllerSessionSigs` is not provided, if `controllerSessionSigs` is not an object, if `executeJsArgs` does not have either `code` or `ipfsId`, or if an error occurs during the execution of the Lit action.
    */
   async runLitAction(toSign: Uint8Array, sigName: string): Promise<any> {
```

Comment:
What's the use of this function?

Reply:
basically just `executeJs`, it's an old function and it needs some love